config.def.h

```diff
@@ -54,45 +61,75 @@ static const Layout layouts[] = {
 #define SHCMD(cmd) { .v = (const char*[]){ "/bin/sh", "-c", cmd, NULL } }
 
 /* commands */
-static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
-static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
+static const char *dmenucmd[] = { "dmenu_run",  NULL };
 static const char *termcmd[]  = { "st", NULL };
+static const char *sLock[]  = { "slock", NULL };
+static const char *backlitInc[]  = { "sudo", "brightnessctl", "set", "+2", NULL };
+static const char *backlitDec[]  = { "sudo", "brightnessctl", "set", "2-", NULL };
+static const char *volToggle[]  = { "pactl", "set-sink-mute", "0", "toggle", NULL };
+static const char *volDown[]  = { "pactl", "set-sink-volume", "0", "-2%", NULL };
+static const char *volUp[]  = { "pactl", "set-sink-volume", "0", "+1%", NULL };
+static const char *audToggle[]  = { "cmus-remote", "-u", NULL };
+static const char *audNext[]  = { "cmus-remote", "-n", NULL };
+static const char *audPrev[]  = { "cmus-remote", "-r", NULL };
+static const char *nnnSpawn[] = { "st", "nnn", NULL };
+static const char *sudonnnSpawn[] = { "sudo", "st", "nnn", NULL };
+static const char *alsamixer[] = { "st", "alsamixer", NULL };
```

Comment:
Is it the simple terminal? If yes then the users who dont use it will have to make changes everywhere. Have a global TERMINAL definition in the config and inject it in functions like this

Reply:
These functions aren't meant to be there always. Since i use this version as my daily driver, I'll need these things. And also, they'll make my job easier. If i remove them now, I'll have to switch between OG dwm and polywm just to check how it's going.

Reply:
You can use a different user on a different tty to test it out